test/superTokenWithdrawHelper.spec.ts

```diff
@@ -0,0 +1,192 @@
+import { constants, Wallet } from "ethers";
+import { assert, contract, web3 } from "hardhat";
```

Comment:
we're using web3 instead of ethers because they use that in their app, right? or is there some other reason?